build/installers/pom.xml

```diff
@@ -36,8 +36,8 @@
         <install-builder-executable>${install-builder-home}/bin/builder</install-builder-executable>
         <gw-latest-download-url>https://github.com/david-truong/gw/releases/download/v1.0.1/gw.jar</gw-latest-download-url>
         <gw-latest-md5>488ddaed413c0f7ea67df049175380eb</gw-latest-md5>
-        <jpmcli-latest-download-url>https://oss.sonatype.org/content/repositories/snapshots/biz/aQute/bnd/biz.aQute.jpm.run/4.0.0-SNAPSHOT/biz.aQute.jpm.run-4.0.0-20201026.162724-24.jar</jpmcli-latest-download-url>
-        <jpmcli-latest-md5>4e53a2098e977cdd69d75be5a661d684</jpmcli-latest-md5>
+        <jpmcli-latest-download-url>http://us-east-1.linodeobjects.com/devtools-s3.liferay.com/liferay-ide-files/biz.aQute.jpm.run-4.0.0-SNAPSHOT.jar</jpmcli-latest-download-url>
```

Comment:
We should switch to `https` version.

Reply:
Also, we should switch to non-SNAPSHOT.jar version.  So the version should be in the jar, otherwise we would never know what the correct version it.  I'll try to release a `4.0.0` version.